### ISSUE TITLE
Accessibility Bug: The label ‘Human Capital Management’ is not visible fully in the normal size and no tooltip also provided.

### ISSUE THREAD

**Issue by puru-hk** [SAP/fundamental-ngx#3148]
Fundamental core component: "Product Switch"

Description: The label ‘Human Capital Management’ is not visible fully in the normal size and no tooltip also provided.

Expected: All the labels should be visible fully in the default size without any truncation.

Attached screenshot:

![image](https://user-images.githubusercontent.com/32538291/91651603-6354d580-eaac-11ea-9f58-129159f46631.png)

**Comment by olshansky** [comment]
Looks fixed:
![image](https://user-images.githubusercontent.com/14160094/100856479-24139780-3494-11eb-99f4-671b691e8f05.png)
